### PR TITLE
Footer bug

### PR DESCRIPTION
**Pull request by kachwalUttamsharma**

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -46,9 +48,19 @@ footer {
...
+  background-color: #041484;
+  color: #f4f4f4;
+  font-weight: 100;
+  margin-top: 10px;
+}
+
+@media only screen and (min-width: 900px) {
+  footer {
+    position: absolute;
+  }
```
Comment (RohitKuwar) [inline styles/globals.css:61]: @kachwalUttamsharma Can you please tell me why are we using this?

**Review by RohitKuwar — COMMENTED** [review]
@kachwalUttamsharma Footer is not responsive.

Screenshot (374)

File: styles/globals.css (hunk 1 above)
Comment (kachwalUttamsharma) [reply styles/globals.css:61]: I used it to make responsive, in between let me re-verify it again.

**Review by RohitKuwar — APPROVED** [review]
@kachwalUttamsharma Approved.

**Review by Kratika0907 — APPROVED** [review]

**Pull request merged**